### ISSUE TITLE
Reporting to JSON format?

### ISSUE THREAD

**Issue by comat0zz** [Endava/cats#22]
Will the report be implemented in the JSON format?
There is not enough for uploading into the defectdojo. At the moment we have to convert html to json an additional script that parses html :cry:

**Comment by en-milie** [comment]
Something like this:

```json
{
  "summaryList": [
    {
      "scenario": "Send [values containing zalgo text] in request fields: field [token_no_default_policy], value [REPLACE with c̷̈́̉̐͂͒̆͂̿͘...], is required [FALSE]",
      "result": "success",
      "resultReason": "Unexpected behaviour",
      "id": "Test 1911",
      "fuzzer": "ZalgoTextInStringFieldsSanitizeValidateFuzzer",
      "path": "/roles/{role_name}"
    },
    {
      "scenario": "Send [values containing zalgo text] in request fields: field [renewable], value [REPLACE with c̷̈́̉̐͂͒̆͂̿͘...], is required [FALSE]",
      "result": "success",
      "resultReason": "Unexpected behaviour",
      "id": "Test 1914",
      "fuzzer": "ZalgoTextInStringFieldsSanitizeValidateFuzzer",
      "path": "/roles/{role_name}"
    }
  ],
  "totalTests": 1341,
  "success": 1333,
  "warnings": 0,
  "errors": 8,
  "executionTime": "8",
  "timestamp": "Wed, 12 Jan 2022 20:25:42 +0200",
  "catsVersion": "7.0.2-SNAPSHOT"
}

```

**Comment by comat0zz** [comment]
Yes, I mean a report in the form of a single JSON file with all the tests.

**Comment by en-milie** [comment]
Ok. The above JSON will be available in 7.0.2 alongside the HTML one.

**Comment by comat0zz** [comment]
Thx )
